src/org/ods/component/Context.groovy

```diff
@@ -672,4 +671,21 @@ class Context implements IContext {
       config.globalExtensionImageLabels.putAll(extensionLabels)
     }
   }
+  
+  public Map<String,String> getExtensionBuildParams () {
+    String rawEnv = script.sh(
+        returnStdout: true, script: "env | grep ods.build.",
+        label: 'getting extension environment labels'
+      ).trim()
+    
+    if (rawEnv.trim().size() == 0 ) {
+      return [:]
+    }
+      
+    return rawEnv.normalize().split(System.getProperty("line.separator")).inject([ : ] ) { kvMap, line ->
+        Iterator kv = line.toString().tokenize("=").iterator()
+        kvMap.put(kv.next(), kv.hasNext() ? kv.next() : "")
+        kvMap
+      }
```

Comment:
Is there any possibility that those lines contain more than one `=` sign?

Reply:
as we own those ods.build guys, I cannot think of one .. (and the worst thing that could happen is that we cut half of it off)